### ISSUE TITLE
EllipticCurveBackend

### ISSUE THREAD

**Issue by rymkapro** [imachug/sslcrypto#10]
Hi!

In some code, I need to get `curve.n` value.

In `fallback.ecc.EllipticCurveBackend` there is attribute `n`, but in `openssl.ecc.EllipticCurveBackend` n attribute is named `order`.

Is it ok? Should I check which backend instance I use?

**Comment by imachug** [comment]
Accessing `_backend` directly is not recommended because it's, uh, backend-related and its internals may change over the time. I have just published v5.3 which allows you to get curve parameters like this:

```python
import sslcrypto
curve = sslcrypto.ecc.get_curve("secp256k1")
assert curve.params["n"] == 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141
```

`p`, `n`, `a`, `b` and `g` (with the latter one being a tuple of two integers) are the supported keys.

Please close the issue if this fix satisfies your needs.

**Comment by rymkapro** [comment]
Thanks once again! Everything is great!
